### PR TITLE
Feature login responsive

### PR DESCRIPTION
**Pull request by DiegoCuevas**
This pull request  contain: 
*  Add responsive for login view

### CODE REVIEW

File: src/view/LoginView.js [hunk 1]
```diff
@@ -11,13 +11,22 @@ function LoginView() {
   const [error, setError] = React.useState(false);
   const { setUser } = useUser();
 
+  const breakpoints = [320, 375, 425, 768];
+  const mq = breakpoints.map(bp => `@media (max-width: ${bp}px)`);
```
Comment (CesarCachay) [inline src/view/LoginView.js]: Could you explain this line please?

Comment (cristianbgp) [reply src/view/LoginView.js]: Could you explain it to me too?

Comment (DiegoCuevas) [reply src/view/LoginView.js]: I put the size of the half queries in a const and I do an iteration of each one and it depends on the size where the app is. In this link you can see what I did (https://emotion.sh/docs/media-queries)

Comment (cristianbgp) [reply src/view/LoginView.js]: Thanks, that's a cool way to use them.

**Pull request merged**